erpnext/stock/utils.py

```diff
@@ -180,12 +180,27 @@ def get_bin(item_code, warehouse):
 	bin_obj.flags.ignore_permissions = True
 	return bin_obj
 
+def get_or_make_bin(item_code, warehouse):
```

Comment:
```suggestion
def get_or_make_bin(item_code, warehouse) -> str:
```

Add some indication that this returns bin name instead of bin object like the function above.